### PR TITLE
[DRAFT] Introduction/Background

### LINKED ISSUE #82: add short high-level background in introduction (e.g. chromatography)
few sentences so that the reader with no background can at least get an idea how to classify the information/topic that follows

### PR DESCRIPTION
**Pull request by oliveralka**
First draft to add a short introduction/background for users new to the field.

EDIT:

- [X] add figures (introduction_LC.png, introduction_MS.png)
- [x] add explanation MS1 & MS2 (see figure introduction_MS.png)

fixes #82

### CODE REVIEW

**Comment by oliveralka** [comment]
Thanks, what do you think? Anything else missing?

File: docs/source/introduction.rst [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+============
+
+pyOpenMS is a python library for Liquid Chromatography-Mass Spectrometry (LC-MS) data analsis.
+
+This introduction is aimed at users new to the field of LC-MS data analysis and will cover some basics in terms of data generation.
+
+How to handle the data analysis, available data structures, algorithms and more is covered in the various subsections of this documentation.
+
+Liquid Chromatography
+---------------------
```
Comment (timosachsenberg) [inline docs/source/introduction.rst:11]: can you run this through a spell checker?

Comment (oliveralka) [reply docs/source/introduction.rst:11]: Good call, forgot about it!

**Comment by timosachsenberg** [comment]
thanks!

**Pull request merged**